### PR TITLE
[Bug Fix] LPS-151508

### PR DESCRIPTION
**Pull request by yuhai**
@dantewang, @tinatian, the pr is based on https://github.com/liferay-core-infra/liferay-portal/pull/655.

I added the [comment](https://issues.liferay.com/browse/LPS-151508?focusedCommentId=2665580&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-2665580) in LPS-151508 to explain the "HOLDLOCK".

### CODE REVIEW

File: modules/apps/view-count/view-count-test/src/testIntegration/java/com/liferay/view/count/service/impl/test/ViewCountEntryLocalServiceTest.java [hunk 1]
```diff
@@ -79,7 +79,8 @@ public void testLazyCreationWithRaceCondition() throws Throwable {
 
 		Assume.assumeFalse(
 			"HSQL does not allow concurrent Session assess, skip test.",
-			db.getDBType() == DBType.HYPERSONIC);
+			(db.getDBType() == DBType.HYPERSONIC) ||
```
Comment (dantewang) [inline modules/apps/view-count/view-count-test/src/testIntegration/java/com/liferay/view/count/service/impl/test/ViewCountEntryLocalServiceTest.java:82]: @yuhai You should put this in another assume statement, as the original assume is for HSQL.

**Comment by yuhai** [comment]
ci:test:stop

**Comment by yuhai** [comment]
ci:stop

**Comment by yuhai** [comment]
ci:stop

**Pull request closed without merging**